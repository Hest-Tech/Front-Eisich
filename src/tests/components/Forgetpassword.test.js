import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

it('render correctly  the forgot password modal', () => {
    const passwordReset = renderer.create(<forgotPassword />).toJSON();
    expect(passwordReset).toMatchSnapshot();

});
