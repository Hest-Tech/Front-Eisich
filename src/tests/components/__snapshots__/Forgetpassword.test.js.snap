// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly  the forgot password modal 1`] = `<forgotPassword />`;
